src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/Runners/MonitorRunner.cs

```diff
@@ -72,6 +72,7 @@ public MonitorRunner(ITestOutputHelper outputHelper)
             // the correct ASP.NET Core version (which can be different than the .NET
             // version, especially for prereleases).
             _runner.FrameworkReference = DotNetFrameworkReference.Microsoft_AspNetCore_App;
+            _runner.TargetFramework = TargetFrameworkMoniker.Net60;
```

Comment:
Rather than attempting to use the current TFM with dotnet-monitor (since that won't work if the test application is running as 3.1 or 5.0), always have dotnet-monitor run on .NET 6. We can adjust this later when testing roll-forward for .NET 7+.